README.md

```diff
@@ -146,6 +146,10 @@ For a debug build:
 Test
 ----
 
+If a ruby was found during configuration, execute the following command before running tests:
```

Comment:
I suspect we could create a `bundle install` target that the `test` target depends on, and avoid the need for a human being to do this step.